opensquirrel/ir.py

```diff
@@ -13,6 +13,15 @@
 from opensquirrel.common import ATOL, are_matrices_equivalent_up_to_global_phase, normalize_angle
 
 
+_REPR_DECIMALS = 5
+
+
+def _repr_round(
+        value: float | Axis | NDArray[np.complex64],
+        decimals: int = _REPR_DECIMALS) -> float | Axis | NDArray[np.complex64]:
+    return np.round(value, decimals)
+
+
```

Comment:
Can you please remove the leading underscores? I would use them as a hint to the reader that a class member/method is intended to be private, but not here.